### ISSUE TITLE
Register kops labels and annotations

### ISSUE THREAD

**Issue by sftim** [kubernetes/kops#16082]
/kind feature

> Kubernetes reserves all labels and annotations in the kubernetes.io and k8s.io namespaces.

&mdash;https://kubernetes.io/docs/reference/labels-annotations-taints/

This is a request to register (and document) all of the labels and annotations that are used in released versions of kops.

I expect this would be a tracking / umbrella issue.

**Comment by justinsb** [comment]
Good point @sftim - we predate a lot of this guidance, but we can definitely add our labels & annotations to the documentation.

**Comment by sftim** [comment]
A detail: kops stabilized (1.4 release) at around the same time that guidance originally appeared; it could have been covered as part of the stabilization work, but got missed - understandably.

Even so, we should register (and document) these annotations.

**Comment by sftim** [comment]
I'll count this one as accepted, then.
/triage accepted
